doc/api/timers.md

```diff
@@ -137,10 +169,11 @@ cancel the timer and prevent it from triggering.
 added: v0.9.1
 -->
 
-* `immediate` {Immediate} An `Immediate` object as returned by
+* `immediate` {Immediate|Promise} An `Immediate` object as returned by
   [`setImmediate()`][].
 
-Cancels an `Immediate` object created by [`setImmediate()`][].
+Cancels an `Immediate` object created by [`setImmediate()`][] or the promisified
+variant of [`setImmediate()`][].
```

Comment:
What happens if a `Promise` that is *not* a promisified-variant of `setImmediate()`?